apps/gui/gui-1.py

```diff
@@ -95,7 +95,10 @@ def _run( self, args ) :
 				GafferUI.FileMenu.addRecentFile( self, fileName )
 				del scriptNode
 		else :
-			self.root()["scripts"].addChild( Gaffer.ScriptNode() )
+			scriptNode = Gaffer.ScriptNode()
+			Gaffer.NodeAlgo.applyUserDefaults( scriptNode )
+			self.root()["scripts"].addChild( scriptNode )
+			del scriptNode
```

Comment:
Good question. I have to admit I put it there out of superstition after seeing the existing `del` just above it in the same file. I've now got to the bottom of why that was there, and removed it in 1f9248c - there's some further explanation in the commit message.